cogs/memes.py

```diff
@@ -151,17 +151,22 @@ def __init__(self, bot):
         await ctx.message.delete()
 
     @commands.command()
-    async def boot(self, ctx, num: int = 2):
-        """Draws a pyramid of boots, default is 3 unless user specifies
+    async def boot(self, ctx, num: int = 2, emoji: str = "👢"):
```

Comment:
You could give this command a more generic alias too, such as `pyramid`. See [here](https://github.com/idoneam/Canary/blob/dev/cogs/reminder.py#L187) for an example on how to add aliases to a command.